department-of-veterans-affairs/caseflow
# Appellant Substitution | Show Alert on Source Appeal

Connects [CASEFLOW-1663](https://vajira.max.gov/browse/CASEFLOW-1663)

### Description
This updates the Case Details page (on source appeal) to display an alert in the "About the Veteran" box if a substitution has taken place

### Acceptance Criteria
- [x] Alert shows in "About the Veteran" with working link to target/substitution appeal

### Testing Plan
1. Go to Case Details page for an appeal _that has had a substitution performed_ (the source appeal)
2. Check the "About the Veteran" section and confirm that `Alert` shows
3. Click on link in the `Alert` and confirm it links to the target/substitution appeal

### Updated Screenshot
![Screen Shot 2021-06-25 at 11 54 15](https://user-images.githubusercontent.com/2581274/123452148-219d9a00-d5ac-11eb-8ced-68b330c14eb8.png)
